core/spec/services/spree/export/google_rss_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'spec_helper'
+require 'pry'
+
+module Spree
+  describe Export::GoogleRss do
+    subject { described_class.new }
+
+    let(:store) { create(:store) }
+    let(:setting) { create(:google_feed_setting, store: store) }
+    let(:product) { create(:product, stores: [store]) }
+    let!(:variant) { create(:with_image_variant, product: product) }
+    let(:result) { subject.call(setting) }
+
+    context 'store header is generated correctly' do
+      before do
+        allow(subject).to receive(:store).and_return(store)
+      end
+
+      it 'store name' do
```

Comment:
```suggestion
      it 'includes store name' do
```
it 'etc etc etc' should always read as a sentence, to make it easy for future devs to know what the test is doing. 

Same for tests below.